File: MedLabO UI/medlabo_desktop/lib/models/rezultat/rezultat.dart
import 'package:json_annotation/json_annotation.dart';

part 'rezultat.g.dart';

@JsonSerializable()
class Rezultat {
  String? rezultatID;
  String? dtRezultata;
  String? testZakljucak;
  bool? obiljezen;
  double? rezFlo;
  String? rezStr;
  double? razlikaOdNormalne;

  Rezultat({
    this.rezultatID,
    this.dtRezultata,
    this.testZakljucak,
    this.obiljezen,
    this.rezFlo,
    this.rezStr,
    this.razlikaOdNormalne,
  });

  factory Rezultat.fromJson(Map<String, dynamic> json) =>
      _$RezultatFromJson(json);

  Map<String, dynamic> toJson() => _$RezultatToJson(this);
}

File: MedLabO.Models/Rezultat.cs
using System.ComponentModel.DataAnnotations;

namespace MedLabO.Models
{
    public class Rezultat
    {
        public string? RezultatID { get; set; }
        public DateTime? DTRezultata { get; set; }
        [Required]
        public string TestZakljucak { get; set; }
        public bool? Obiljezen { get; set; } = false;
        public float? RezFlo { get; set; }
        public string? RezStr { get; set; }
        public float? RazlikaOdNormalne { get; set; }
    }
}
